api/src/utils/validate-primary-keys.ts

```diff
@@ -0,0 +1,27 @@
+import { SchemaOverview } from '@directus/shared/types';
+import { ForbiddenException } from '../exceptions';
+import { PrimaryKey } from '../types';
+import validateUUID from 'uuid-validate';
+
+/**
+ * Validate primary keys based on its type
+ */
+export function validatePrimaryKeys(
+	schema: SchemaOverview,
+	collection: string,
+	primaryKeyField: string,
+	keys: PrimaryKey | PrimaryKey[]
```

Comment:
Yes, you are right about the possibility to simplify the type signature. However, that may limit the usage of this util. The current version allows for usage on checking keys of relational fields, by passing in the relevant `collection` and `field`.

What do you think? Or should I rename the util to be something more generic along the lines of `validate-field-keys-type`?

Reply:
Ahh gotcha! Yeah lets rename it to be more generic then (like `validateKeys`), and instead of calling the argument `primaryKeyField`, lets call it `keyField` 👍🏻

Reply:
Done. Thanks! 🤓